nncf/common/pruning/mask_propagation.py

```diff
@@ -94,7 +98,7 @@ def symbolic_mask_propagation(self, prunable_layers_types: List[str],
                 node.data['output_mask'] = SymbolicMask(node.layer_attributes.out_channels, [node.node_id])
             # Propagate masks
             cls = self.get_meta_operation_by_type_name(node.node_type)
-            cls.mask_propagation(node, self._graph)
+            cls.mask_propagation(node, self._graph, self._tensor_processor)
```

Comment:
Here only Symbolic mask processor can be used. I suggest to specify Symbolic mask processor here explicitly and make `self._tensor_processor` attribute optional in case only symbolic mask propagation is used

Reply:
```suggestion
            cls.mask_propagation(node, self._graph, SymbolicMaskProcessor)
```